ludwig/schema/features/preprocessing/image.py

```diff
@@ -109,12 +111,12 @@ class ImagePreprocessingConfig(BasePreprocessingConfig):
         parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["infer_image_sample_size"],
     )
 
-    scaling: str = schema_utils.StringOptions(
-        ["pixel_normalization", "pixel_standardization"],
-        default="pixel_normalization",
-        allow_none=False,
-        description="The scaling strategy for pixel values in the image.",
-        parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["scaling"],
+    standardize_image: Union[str, None] = schema_utils.StringOptions(
+        [None, "imagenet1k"],
```

Comment:
Small nit: I would still remove `None` from the options list since `allow_none=True`, unless you mean to emphasize that for a different reason.

Reply:
will do.